src/icons.tsx

```diff
@@ -19,6 +22,12 @@ export const Icons: React.FC<IconsProps> = ({ icon, className }) => {
   if (icon === ToolbarButtonType.CreateArrow) {
     return <CreateArrow className={className} />;
   }
+  if (icon === ContextMenuButtonType.Edit) {
```

Comment:
`Icons` could PERHAPS be re-written as 

```tsx
const allIcons = {
  [ToolbarButtonType.MapColor]: <MapColor className={className} />,
  [ToolbarButtonType.CreateBox]: <CreateBox className={className} />,
  [ToolbarButtonType.CreateArrow]: <CreateArrow className={className} />,
  [ToolbarButtonType.Edit]: <Edit className={className} />,
  [ToolbarButtonType.Delete]: <Delete className={className} />,
};


export const Icons: React.FC<IconsProps> = ({ icon, className }) => {
  const defaultIcon = <MapColor className={className} />;
  
  return allIcons[icon] ?? defaultIcon;
};
```

Reply:
(and be called "Icon"?)

Reply:
How will className be included? Maybe have the change with icons in a new task?